.jenkins/pytorch/common_utils.sh

```diff
@@ -106,3 +106,7 @@ function clone_pytorch_xla() {
     popd
   fi
 }
+
+function install_torchdynamo() {
+  pip_install --user "git+https://github.com/pytorch/torchdynamo.git@$(cat .github/torchdynamo_commit_hash.txt)"
```

Comment:
Where does `.github/torchdynamo_commit_hash.txt` come from?

Reply:
this: https://github.com/pytorch/pytorch/pull/79042 it's hard coded and gets a PR to be updated nightly